### PR TITLE
upgrade cloud plugin versions for 4.0 release

### PR DESCRIPTION
**Pull request by hasancelik**

### CODE REVIEW

**Comment by mmedenjak** [comment]
Can you send a forward-port against master? Thanks!

File: hazelcast-all/pom.xml [hunk 1]
```diff
@@ -149,29 +149,26 @@
...
-        <!-- Make sure that the core module is the first dependency! -->
-        <!--        https://github.com/hazelcast/hazelcast/issues/15576-->
-        <!--dependency>
+        <dependency>
             <groupId>com.hazelcast</groupId>
             <artifactId>hazelcast-aws</artifactId>
-            <version>2.4</version>
-        </dependency-->
+            <version>3.0</version>
+        </dependency
```
Comment (Holmistr) [inline hazelcast-all/pom.xml]: Missing bracket here, thus not compiling.

Comment (hasancelik) [reply hazelcast-all/pom.xml]: fixed 👍

**Comment by hasancelik** [comment]
@mmedenjak Here is the PR for master --> https://github.com/hazelcast/hazelcast/pull/15747

**Pull request merged**